deep_speech_2/tune.py

```diff
@@ -0,0 +1,238 @@
+"""Parameters tuning for DeepSpeech2 model."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import paddle.v2 as paddle
```

Comment:
1. Reorder the imports.
2. Please modify all below according to the suggestions in infer.py and evaluate.py.
3. Add descriptions to README.md for usage of tune.py and evaluate.py.